cmd/web/web.go

```diff
@@ -63,8 +74,20 @@ func NewWebCmd() *cobra.Command {
 	})
 
 	webCmd.AddCommand(serveCmd)
+}
 
-	return webCmd
+func addPruneCmd(webCmd *cobra.Command) {
+	var olderThan int
+
+	pruneCmd := &cobra.Command{
+		Use:   "prune-events",
+		Short: "Prune events older than",
+		Run:   prune,
+	}
+
+	pruneCmd.Flags().IntVar(&olderThan, "older-than", 10, "Prune events older than (in days, required)")
```

Comment:
This should not be required anymore

Reply:
I would actually be more precise on this. Something like:
`Prune auto discovered events (older than x days)`
Yes, it is difficult to define this flag definition hehe